contracts/beacon/Voucher.sol

```diff
@@ -233,12 +306,60 @@ contract Voucher is OwnableUpgradeable, IVoucher {
         $._authorizedAccounts[account] = isAuthorized;
     }
 
+    /**
+     * Ask VoucherHub to refund voucher for a failed task and
+     * send non-sponsored part back to the requester when needed.
+     * @param voucherHub hub
+     * @param iexecPoco address of PoCo contract
+     * @param taskId id of the task
+     * @param dealId task's deal id
+     * @param dealVolume number of tasks in the deal
+     * @param dealPrice price paid at match orders
+     * @param requester of the task
+     */
+    function _refundVoucherAndRequester(
+        IVoucherHub voucherHub,
```

Comment:
Is it cheaper to pass the hub instead of reading `$._voucherHub` inside this current method?

Reply:
Cheaper yes:
* Classic ~160 gas
* Boost ~130 gas

Reply:
Nice thanks